packages/react/src/FlowMap.tsx

```diff
@@ -67,7 +67,7 @@ export default class FlowMap extends React.Component<Props, State> {
   };
 
   static getDerivedStateFromProps(props: Props, state: State): Partial<State> | null {
-    if (props.selectedLocationIds && props.selectedLocationIds !== state.selectedLocationIds) {
+    if (typeof(props.selectedLocationIds) !== 'undefined' && props.selectedLocationIds !== state.selectedLocationIds) {
```

Comment:
I think it's safe to simply drop the first part of the condition:
```suggestion
    if (props.selectedLocationIds !== state.selectedLocationIds) {
```